src/js/modules/sections/components/SectionPage.js

```diff
@@ -3,26 +3,121 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import injectSheet from 'react-jss';
-import sections from '../../sections';
+import { Grid, Row, Col } from 'react-bootstrap/lib/';
 
 import articles from '../../articles';
 
 const styles = {
   SectionPage: {
     color: '#000',
+  },
```

Comment:
The naming for this JSS is not optimal. From each class name, I should be able to tell the part of the page it styles. Change the names that aren't descriptive (including `Horizontal`, `linkColor`, and `image`.)